euler_support/urdf/euler_macro.xacro

```diff
@@ -1,19 +1,17 @@
 <?xml version="1.0" ?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
+    <xacro:macro name="euler" params="prefix">
+        <xacro:include filename="$(find motoman_mh5lf_support)/urdf/mh5lf_macro.xacro" />
+        <xacro:motoman_mh5lf prefix="${prefix}motoman_" />
+        <!--xacro:include filename="$(find euler_support)/urdf/asus_xtion_pro.xacro" /-->
```

Comment:
Remove commented line.

Reply:
You want this deleted or uncommented?

Reply:
Deleted unless you find that it's necessary to make the sensor show.